lldb/source/Plugins/LanguageRuntime/Swift/ReflectionContext.cpp

```diff
@@ -368,6 +369,28 @@ class TargetReflectionContext : public ReflectionContextInterface {
     return m_reflection_ctx.stripSignedPointer(pointer);
   }
 
+  std::pair<std::optional<std::string>, AsyncTaskInfo>
```

Comment:
This return type looks like poor man's `llvm::Expected<AsyncTaskInfo>` returning an `llvm::createStringError()`?